Doc/library/ctypes.rst

```diff
@@ -147,6 +147,12 @@ functions can be accessed by indexing the dll object with the ordinal number::
 Calling functions
 ^^^^^^^^^^^^^^^^^
 
+.. deprecated-removed:: 3.6.2 3.7
+
+   Calling :mod:`ctypes` functions is deprecated. Calling a native function
+   with incorrect arguments is unsupported and can easily cause
+   information leakage or code execution vulnerabilities.
```

Comment:
This comment suggests that `ctypes` functions cannot be called, i.e. that the `GetModuleHandleA()` call in the next paragraph will soon be unsupported.

Perhaps move this deprecation warning down below the "`ctypes` tries to protect you from calling functions with the wrong number of arguments" paragraph (line 164/170) and reword as "This protection against incorrect arguments is deprecated and will be removed."